src/styles/card.scss

```diff
@@ -0,0 +1,99 @@
+@import './variables';
+@import './stars';
+
+.card__link {
+  box-sizing: border-box;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 40px;
```

Comment:
it's not a good practice to hardcode height fir elements with content inside. add padding instead